### ISSUE TITLE
Add a mean to review and comment blog posts

### ISSUE THREAD

**Issue by ctmbl** [iScsc/iscsc.fr#77]
Currently blocked by #12 

## Problem
There is no mean, at the time, to review blog posts, for example to fix typos, add/ask for additional information or warn about dead links.  
We should also be able to report blog posts.

## Some Solutions
Here is a list (from the simplest to the most complex ones) of solutions I imagined:
 - a simple comment section below the post (this is for another issue but the author could be notified)
 - a mean to send private messages to/email the author, with or without some embedding to highlight/show the `diff` one would like
 - a more complex workflow where, after submitting a post an author, the post is in `Review` state (analogous to #75) and the author must wait for, designated or not, reviewers approval (same as GitHub PRs or Root-Me solution submitting)
 - some other I didn't think of ??

**Comment by ctmbl** [comment]
@atxr 
Thanks for the answer!

I'd like comments section so I'm happy you prefer it too.
However we should first make the blog posts editable don't you think?
